internal/testutils/specialimage/multilayer.go

```diff
@@ -53,7 +53,7 @@ func MultiLayer(dir string) error {
 	}
 
 	legacyManifests := []manifestItem{
-		manifestItem{
+		{
```

Comment:
Oh! Good that I see this; I just noticed earlier that `gofmt` gave me a diff in this file, and "go report" showed it as well; https://goreportcard.com/report/github.com/docker/docker